lib/bank_accounts/account.rb

```diff
@@ -0,0 +1,45 @@
+require 'pry'
+
+module BankAccounts
+  class Account
+
+    attr_reader :id, :balance
+
+    # Rounds float value down to hundredths position
+    # For example 10.2599 rounds to 10.25
+    def round_to_hundredths(n)
+      n = (((n * 100).floor) / 100.0)
+      return n
```

Comment:
It probably varies developer to developer. I've noticed that in Seattle there is a tendency to not use more characters than you need. So using implicit returns save a whole line. Additionally for more complex methods, you can simply have the last line of the method be the variable that gets returned. The word `return` is extra unnecessary. It is a small thing. I was noticing other folks were discussing implicit returns and didn't want you to miss out on that feedback but I don't think it is hugely important either. Most importantly, when you are reading code, understanding implicit returns will be very helpful.